styles.css

```diff
@@ -102,9 +105,294 @@ a:link { text-decoration: none; }
 }
 
 .bottom-icons li {
-  padding: 5px;
+  padding: 10px;
 }
 
 .bottom-icons img {
   height: 20px;
 }
+
+.cards {
+  box-sizing: border-box;
+  display: grid;
+  background: #e5e5e5;
+  min-height: 100vh;
+  grid-template-columns: 1fr;
+  grid-template-rows: auto;
+  row-gap: 114px;
+  padding-left: 24px;
+  padding-right: 24px;
+  padding-top: 114px;
+}
+
+.project1 {
+  display: flex;
+  align-items: center;
+  padding-top: 16px;
+  padding-bottom: 30px;
+  flex-direction: column;
+  background-color: #fff;
+  border-style: solid;
+  border-width: 1px;
+  border-color: #dfe1e6;
+  border-radius: 16px;
+}
+
+.tonic-title {
+  display: flex;
+  justify-content: end;
+  margin-left: -24px;
+}
+
+h2 {
+  width: 295px;
+  font-family: 'Poppins', 'Times New Roman', Times, serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  flex-grow: 0;
+  margin-top: 0;
+  margin-left: 19px;
+  margin-bottom: 0;
+}
+
+.tonic-info {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 70%;
+  margin-left: 14px;
+  padding-left: 7px;
+  padding-right: 80px;
+  font-family: 'Poppins', 'Times New Roman', Times, serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+}
+
+.tonic-span {
+  color: #344563;
+}
+
+.tonic-p {
+  color: #7a869a;
+}
+
+.project-p {
+  font-family: 'Poppins', 'Times New Roman', Times, serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  margin-left: 0;
+  margin-top: -20px;
+  width: 295px;
+  height: 76px;
+  margin-bottom: 5px;
+}
+
+.skills {
+  display: flex;
+  width: 100%;
+}
+
+.skills ul {
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  margin-right: 140px;
+  padding-left: 20px;
+}
+
+.skills li {
+  font-family: 'Poppins', 'Times New Roman', Times, serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  padding: 4px;
+  background: #ebebff;
+  border-style: solid;
+  border-width: 1px;
+  border-color: #6070ff;
```

Comment:
- [ ] Kindly remove the border of the skills in your design to look similar to the Figma design. 👇🏽 

- **Your design**
![image](https://user-images.githubusercontent.com/58642949/192077486-88e51093-6719-4733-a4a1-012a278dfc2a.png)

- **Figma design**
![image](https://user-images.githubusercontent.com/58642949/192077508-d5afb31d-20a4-48a6-b5a9-a8b9a3bbb63b.png)